sdk/tests/deqp/functional/gles3/es3fFboTestUtil.js

```diff
@@ -283,7 +283,7 @@ es3fFboTestUtil.FboIncompleteException.prototype.getReason = function() {return
             /** @const {number} */ var f1 = 0.5 + (x - y) * 0.5;
             /** @const {Array<number>} */ var fv = [f0, f1, 1.0 - f0, 1.0 - f1];
 
-            /** @const {Array<number>} */ var color = deMath.clampVector(deMath.add(gradientMin, deMath.multiply(deMath.subtract(gradientMax, gradientMin), fv)), 0, 1);
+            /** @const {Array<number>} */ var color = deMath.add(gradientMin, deMath.multiply(deMath.subtract(gradientMax, gradientMin), fv));
```

Comment:
https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/gles3/functional/es3fFboTestUtil.cpp#222
color is always in [min, max] and needn't to clamp to [0, 1]. The same below.